app/views/dashboard/works/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<%= content_for :navbar_items do %>
+  <li class="nav-item">
+    <%= link_to(t('.back'),
+                resource_path(@work.uuid),
+                class: 'btn btn-outline-light btn--squish mr-lg-2 my-1') %>
+  </li>
+<% end %>
+
+<%= content_for :body_attributes, 'data-spy="scroll" data-target="#navbar-settings" data-offset="10" style="position: relative"'.html_safe %>
```

Comment:
This is needed for scrollspy. We have to add these attributes to the `<body>` tag